src/test/rest_utils_test.cpp

```diff
@@ -1099,6 +1255,71 @@ TEST_F(KFSMakeJsonFromPredictResponseValTest, MakeJsonFromPredictResponse_Positi
 })");
 }
 
+TEST_F(KFSMakeJsonFromPredictResponseValTest, MakeJsonFromPredictResponse_Positive_oneOutputsBinary) {
+    std::set<std::string> binaryOutputs;
+    binaryOutputs.insert("single_uint64_val");
+    ASSERT_EQ(makeJsonFromPredictResponse(proto, &json, inferenceHeaderContentLength, binaryOutputs), StatusCode::OK);
+    ASSERT_EQ(inferenceHeaderContentLength.has_value(), true);
+    std::string expectedJson = R"({
+    "model_name": "model",
+    "id": "id",
+    "outputs": [{
+            "name": "single_uint64_val",
+            "shape": [1],
+            "datatype": "UINT64",
+            "parameters": {
+                "binary_data_size": 8
+            }
+        }, {
+            "name": "two_uint32_vals",
+            "shape": [2],
+            "datatype": "UINT32",
+            "data": [4000000000, 1]
+        }]
+})";
+    ASSERT_EQ(inferenceHeaderContentLength.value(), expectedJson.size());
+    ASSERT_EQ(json.size(), expectedJson.size() + sizeof(uint64_t));
+    EXPECT_EQ(json.substr(0, inferenceHeaderContentLength.value()), expectedJson);
+    uint64_t expectedData = 5000000000;
+    EXPECT_EQ(*(uint64_t*)json.substr(inferenceHeaderContentLength.value()).data(), expectedData);
```

Comment:
Could assertDataBinary be used here?

Reply:
assertDataBinary is method from different test class, i dont think it is worth it to add this method to KFSMakeJsonFromPredictResponseValTest for this case only.

Reply:
Could we use inheritence to avoid `assertDataBinary` declaration duplicate?